docs/quickstart.rst

```diff
@@ -76,6 +76,7 @@ than $50,000 a year.
 
     >>> import numpy as np 
     >>> import pandas as pd
+    >>> pd.set_option('display.precision', 4)
```

Comment:
Why don't we keep it as is? Ideally such code shouldn't be in there as it's unrelated to the example.

Reply:
From the PR description 'Something had changed about how Pandas was printing out DataFrame objects'

I'm not sure what had changed, although the recent release of pandas 1.2.0 is a strong suspect. The point is `doctest` does string matching, so it's either do this, or scatter ellipses throughout the results (which then act as wildcards). I believe that we could move directives like this into a hidden code block, but I think I'd prefer to leave the change out in the open.

Reply:
I've mentioned before that we should keep the precision to 3 or 4 and then `...` in the doctests, so I'd say we should go with that and not include this display option in the example.

Reply:
I'm not quite sure what you mean. It appears that pandas has changed its default display for DataFrames. At least some of the updates I had to make involved changing a `0` or `1` to `0.0` or `1.0`. While I suppose those could be handled by the `...` wildcard, I don't think it would look good in the docs.

Reply:
I suppose that, since we don't build the docs with old Pandas versions, I could just update to the new formatting (and truncate decimals at 3 or 4 places).... is that what you meant @adrinjalali ?

Reply:
That's definitely a solution. I'm a bit surprised about 1.0 vs 1 though. That means there's a dtype change between versions? If that's the case, we can force the dtype and make sure it's always int or float, whichever it needs to be.

Reply:
It might also be related to our dtype being 'object' a lot of the time. Anyway, I've just pushed the update.

Reply:
My main point was that I don't want this particular line in all examples, and that's addressed, so I'll sign off happily :-) Thanks!

Reply:
@adrinjalali does this look better to you?